jwst/outlier_detection/tests/test_outlier_detection.py

```diff
@@ -604,14 +607,75 @@ def test_create_median(three_sci_as_asn, tmp_cwd):
     with (lib_on_disk, lib_in_memory):
         for lib in [lib_on_disk, lib_in_memory]:
             for model in lib:
-                model.wht = np.ones_like(model.data)
-                model.wht[4,9] = 0.5
+                model.var_rnoise = np.ones_like(model.data)
+                model.var_rnoise[4,9] = 2.0
                 lib.shelve(model, modify=True)
 
-    median_on_disk = create_median(lib_on_disk, 0.7)
-    median_in_memory = create_median(lib_in_memory, 0.7)
+
+    # 32-bit floats are 4 bytes each, min buffer size is one row of 20 pixels
+    # arbitrarily use 5 times that
+    buffer_size = 4 * 20 * 5 
+    median_on_disk, _ = drizzle_and_median(lib_on_disk,
+                                           make_resamp(lib_on_disk, False),
+                                           0.7,
+                                           buffer_size=buffer_size,
+                                           resample_data=False)
+    median_in_memory, _ = drizzle_and_median(lib_in_memory,
+                                             make_resamp(lib_in_memory, False),
```

Comment:
`in_memory=True` should be passed into `make_resamp` here.  However, unless `resample_many_to_many` is called directly (which it is not during this test), `in_memory` has no effect inside `ResampleData` so this test is meaningful either way.  I fixed and tested this locally but will wait to push it until there are more fixes from other people's reviews